core-web/libs/ui/src/lib/components/dot-drop-zone/dot-drop-zone.component.ts

```diff
@@ -31,15 +34,18 @@ export interface DropZoneFileValidity {
 export class DotDropZoneComponent {
     @Output() fileDropped = new EventEmitter<DropZoneFileEvent>();
     @Output() fileDragEnter = new EventEmitter<boolean>();
+    @Output() fileDragOver = new EventEmitter<boolean>();
     @Output() fileDragLeave = new EventEmitter<boolean>();
 
     @Input() maxFileSize: number;
 
     @Input() set accept(types: string[]) {
-        this._accept = types.map((type) => {
-            // Remove the wildcard character
-            return type.toLowerCase().replace(/\*/g, '');
-        });
+        this._accept = types
+            ?.filter((value) => value !== '*/*')
```

Comment:
Now that we have accept in the binary... I am thinking if we need accept here.

Reply:
Since this component can be used in other places, not only with the binary field, it's still needed.